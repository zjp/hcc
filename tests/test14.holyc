f	12q-f	q9p2fpqojwehc;lkSJdf a
;sklfj
	q234r	l2u3ty6-	28y734t	]
	hasdfjap8923yurtb	

	qwefj[qov234i5	
intptr 1
		b2'5
		9
			q[2p39
			3b120	
			dfadf
