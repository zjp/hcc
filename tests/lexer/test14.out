ID:f [1,1]
INTLIT:12 [1,3]
ID:q [1,5]
DASH [1,6]
ID:f [1,7]
ID:q9p2fpqojwehc [1,9]
SEMICOLON [1,22]
ID:lkSJdf [1,23]
ID:a [1,30]
SEMICOLON [2,1]
ID:sklfj [2,2]
ID:q234r [3,2]
ID:l2u3ty6 [3,8]
DASH [3,15]
INTLIT:28 [3,17]
ID:y734t [3,19]
RBRACE [3,25]
ID:hasdfjap8923yurtb [4,2]
ID:qwefj [6,2]
LBRACE [6,7]
ID:qov234i5 [6,8]
INTPTR [7,1]
INTLIT:1 [7,8]
ID:b2 [8,3]
CHARLIT:5
INTLIT:9 [9,3]
ID:q [10,4]
LBRACE [10,5]
INTLIT:2 [10,6]
ID:p39 [10,7]
INTLIT:3 [11,4]
ID:b120 [11,5]
ID:dfadf [12,4]
EOF [13,1]
